packages/core/src/application_ref.ts

```diff
@@ -734,8 +733,9 @@ export class ApplicationRef {
     });
 
     this._loadComponent(compRef);
-    if (isDevMode()) {
-      this._console.log(
+    if (typeof ngDevMode === 'undefined' || ngDevMode) {
+      const _console = this._injector.get(Console);
```

Comment:
I believe we should retain `isDevMode()` call (to make things work in ViewEngine) here and just add the `ngDevMode` to make `Console` tree-shakable in Ivy:
```suggestion
    if ((typeof ngDevMode === 'undefined' || ngDevMode) && isDevMode()) {
      const _console = this._injector.get(Console);
```
The same pattern is used in the `application_ref.ts` file above, see:
https://github.com/angular/angular/blob/master/packages/core/src/application_ref.ts#L764-L770